core/src/test/java/org/apache/spark/util/collection/unsafe/sort/UnsafeExternalSorterSuite.java

```diff
@@ -101,7 +101,7 @@ public OutputStream apply(OutputStream stream) {
   public void setUp() {
     MockitoAnnotations.initMocks(this);
     sparkConf = new SparkConf();
-    tempDir = new File(Utils.createTempDir$default$1());
+    tempDir = Utils.createTempDir(System.getProperty("java.io.tmpdir"), "unsafe-test");
```

Comment:
Oh right it makes a subdir every time anyway. Well, it'd be better to clean up the temp dir in `tearDown` if possible.